mandatory/1-fix-functions.js

```diff
@@ -10,31 +10,29 @@
 
 */
 
-function mood() {
-  let isHappy = true;
-
-  if (isHappy) {
+function mood(myMood) {
+  if (myMood === isHappy) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
```

Comment:
Hey Niamh, your code is easy to read and understand, full marks :)
 Your approach to this function is really interesting! Did you delete line 14 by accident, since without it isHappy is undefined, and will fail the test?

Reply:
Hey, thanks for the feedback, appreciate it. I’ll go back and make the changes :)